modules/output/json/main.nf

```diff
@@ -172,6 +172,11 @@ def writeProtein(String proteinMd5, Map proteinMatches, JsonGenerator jsonWriter
 
 def writeMatch(String proteinMd5, Map match, JsonGenerator jsonWriter) {
     // Write out an individual match to an array of matches. The structure is dependent on the memberDB.
+    if (match.source == "InterPro-N") {
+        writeInterProN(match, jsonWriter)
+        return
+    }
+
     String memberDB = match.signature.signatureLibraryRelease.library.toLowerCase() ?: ""
```

Comment:
Why not integrate this into the `switch/case` with setting memberDB to `InterPro-N` if `match.source == "InterPro-N" ? "InterPro-N" : match.signature.signatureLibraryRelease.library.toLowerCase() ?: ""`

```suggestion
    String memberDB = match.source == "InterPro-N" ? "InterPro-N" : match.signature.signatureLibraryRelease.library.toLowerCase() ?: ""
```